A/AMGX_jll/Compat.toml

```
@@ -0,0 +1,4 @@
+[2]
+CUDA_jll = "10.0.0"
```

@KristofferC there is no v10.0.0, so this is complaining.  I think this should be
```suggestion
CUDA_jll = "10.0"
```
?

---

Yes

---

One day I'll understand compat in the registry :smile: 

Now it even passes the loading test here, cool!